instaclustr/resource_cluster.go

```diff
@@ -520,6 +520,11 @@ func resourceCluster() *schema.Resource {
 										Optional: true,
 										ForceNew: true,
 									},
+									"redis_password_auth": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										ForceNew: true,
```

Comment:
Does changing this have to force a new resource? Can we add password auth after creating the cluster?

Reply:
We normally provision a new cluster if we want the cluster to be client encrypted